### DISCUSSION TITLE
Can barely boot the computer and occasionally get into it, usually stuck at the following code

### DISCUSSION THREAD

**Discussion by kingliu6** [osy86/HaC-Mini#988] [General]
**I can barely boot the computer and occasionally get into it, usually stuck at the following code, tried many ways but nothing worked out, I would be grateful if you could help  @osy **
![15414368906a3fa73d81e1f11da9d8a](https://user-images.githubusercontent.com/101332459/158139945-a33a2961-4471-4a71-af7f-31fdc408e5b9.jpg)

**Comment by osy** [comment]
Did you install latest version of HaC mini?

Did you follow the support page of the guide?

**Reply by kingliu6** [reply]
Sure

**Reply by kingliu6** [reply]
I have tried many ways

**Reply by anti-decoy** [reply]
I have met the same problem.

**Reply by kingliu6** [reply]
Just remove the DW1820A card

**Reply by anti-decoy** [reply]
Thanks!  It's ok!
